dataprofiler/profilers/order_column_profile.py

```diff
@@ -1,4 +1,9 @@
 """Index profile analysis for individual col within structured profiling."""
+from __future__ import annotations
+
+from typing import Dict, Optional, Tuple, cast
+
+from pandas import DataFrame, Series
 
```

Comment:
this formatting looks a bit off, but maybe it is right. Did the `pre-commit` run when you did `git commit -m`

Reply:
`isort` did run

Reply:
perfect